client/i18n/locales/japanese/links.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Any of the links in this file that have been translated to Japanese can be replaced with the appropriate URL